crml/eth-wallet/src/lib.rs

```diff
@@ -99,9 +99,10 @@ decl_module! {
 			// convert to CENNZnet address
 			let account = T::AddressMapping::into_account_id(eth_address);
 			let nonce = <frame_system::Pallet<T>>::account_nonce(account.clone());
-			let message = &(&call, nonce).encode()[..];
-
-			if let Some(_public_key) = ecrecover(&signature, message, &eth_address) {
+			let prefix = "data:application-octet;base64,";
```

Comment:
why include this?
- should it be `application/octet-stream`?
- use a constant

Reply:
ah yip, should be `data:application/octet-stream;base64,`

@holygits it's a convention when you base64 encode stuff, so the user know the type of the encoded message, also as a little bonus, if you cut n paste that in the browser, it will understand and decode the message too.

Do you reckon it adds too much of tx size overall?

Reply:
cool thing is the prefix is added on both sides (not sent) so it doesn't increase the tx size at all